### PR TITLE
Complete adduser route

### PR DESCRIPTION
**Pull request by jakec-github**
For some reason this commit is modifying package-lock.json. That may be an issue as it is adding ^ to all the version numbers. 

I also had to comment out some console logs in the server.js file in order to get the pre commit hooks to pass. Let me know what I need do to get this approved.

### CODE REVIEW

File: server/controllers/index.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
+// import https from 'https'
+
```
Comment (AnnaVih) [inline server/controllers/index.js:2]: ??

File: server/controllers/index.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 import Katathon from '../models/katathonModel'
 
 import {
   dateToTimestamp,
   getNextEvent
 } from '../helpers'
 
+const https = require('https')
+
```
Comment (AnnaVih) [inline server/controllers/index.js]: Any preferences to require over import?
To be more consistent we can use import everywhere.?

File: server/controllers/index.js [hunk 3]
```diff
@@ -183,7 +187,64 @@ export const nextKatathon = async (req, res) => {
...
+          if (!isNew) {
+            throw new Error('User already in katathon')
+          }
+
+          katathon.users = [newUser, ...katathon.users]
+          katathon.save()
+          res.status(200).json({
+            result: 'Success',
+            data: katathon,
+            message: 'New kata has been added to the Katathon'
```
Comment (AnnaVih) [inline server/controllers/index.js]: new Kata or new user ? ;)

File: server/controllers/index.js (hunk 2 above)
Comment (webtechalex) [reply server/controllers/index.js]: Agreed. Prefer ES2015 import

Comment (jakec-github) [reply server/controllers/index.js]: Ah yes. This should be import.

File: server/controllers/index.js (hunk 3 above)
Comment (jakec-github) [reply server/controllers/index.js]: This should probably be new user then

**Comment by jakec-github** [comment]
Any word on the package-lock.json? Should I just remove it from the commit?

**Comment by webtechalex** [comment]
Don't remove the package-lock! Will comment further when I'm home...

**Comment by webtechalex** [comment]
The issue with the package-lock has likely occurred because one of us had manually updated something in the package.json and not done an npm install then a commit. Leave the change in so that it gets updated for everyone and make sure that you npm install after any manual changes to package.json before you commit the change to avoid this happening in future

**Review by AnnaVih — APPROVED** [review]

**Comment by jakec-github** [comment]
Ok. New commit has been added fixing issues you guys pointed out and I left the updated package-lock.json in

File: server/controllers/index.js (hunk 1 above)
Comment (webtechalex) [reply server/controllers/index.js:2]: @AnnaVih what were you questioning here?

**Pull request merged**